lib/exrm/plugins/conform.ex

```diff
@@ -34,6 +34,24 @@ defmodule ReleaseManager.Plugin.Conform do
         warn "Conform: No schema found, conform will not be packaged in this release!"
         false
       true ->
+        {:ok, curr_path} = File.cwd
+        imports = Keyword.get(merged, :import, [])
+        File.cd("_build/" <> (Mix.env |> Atom.to_string) <> "/lib")
+
+        build_files = Enum.reduce(imports, [], fn(deps_app, acc) ->
+          {:ok, files_list} = :file.list_dir("#{deps_app}/ebin")
+          files_list = Enum.map(files_list, fn(filename) -> ("#{deps_app}/ebin/" <> "#{filename}") |> to_char_list end)
+          :lists.append(acc, files_list)
+        end)
+
+        case build_files do
+          [] ->
+            File.cd(curr_path)
+          _ ->
+            :zip.create("#{curr_path}/rel/#{app}/releases/#{version}/app.schema.deps.ez" |> to_char_list, build_files)
```

Comment:
Yeah I agree with @liveforeverx, should be able to do this by doing something like this:

```
# Add relx_config to pattern match in `before_release` head
def before_release(%Config{name: app, version: version, relx_config: relx_config}) do

# Then create the zip and add the overlay
zip_path         = :zip.create(...)
with_overlays = [{:overlay, [{:copy, '#{zip_path}', 'releases/#{version}/app.schema.deps.ez'}]}]
relx_config     = Utils.merge(relx_config, with_overlays)

# Persist the updated relx.config
Utils.write_terms(Utils.rel_file_dest_path("relx.config"), relx_config)

# Make sure you return the updated config to exrm
%{config | :relx_config => relx_config}
```